lib/v2/ecust/radar.js

```diff
@@ -17,5 +17,13 @@ module.exports = {
                 target: '/ecust/yjs',
             },
         ],
+        jwc: [
+            {
+                title: '教务处信息网',
```

Comment:
```suggestion
                title: '本科教务处信息网',
```
`title` here should match the H3 heading of the docs